packages/coreutils/selinux.patch

```diff
@@ -27,9 +29,52 @@
  
    /* FIXME: This handles new files.  But what about existing files?
       I.e., if updating a tree, new files would have the specified context,
---- coreutils-9.5.orig/src/mkdir.c
-+++ coreutils-9.5/src/mkdir.c
-@@ -228,7 +228,7 @@
+diff --git a/src/install.c b/src/install.c
+index b3b26abdb..c724dbe34 100644
+--- a/src/install.c
++++ b/src/install.c
+@@ -325,6 +325,9 @@ setdefaultfilecon (char const *file)
+   struct stat st;
+   char *scontext_raw = nullptr;
+ 
++  /* FIXME: Return early for now until a suitable workaround has been found */
++  return;
++
+   if (selinux_enabled != 1)
+     {
+       /* Indicate no context found. */
+@@ -789,7 +792,7 @@ main (int argc, char **argv)
+   bool strip_program_specified = false;
+   char const *scontext = nullptr;
+   /* set iff kernel has extra selinux system calls */
+-  selinux_enabled = (0 < is_selinux_enabled ());
++  selinux_enabled = (0 < is_selinux_enabled () && geteuid () == 0);
+ 
+   initialize_main (&argc, &argv);
+   set_program_name (argv[0]);
+@@ -876,7 +879,7 @@ main (int argc, char **argv)
+           if (! selinux_enabled)
+             {
+               error (0, 0, _("WARNING: ignoring --preserve-context; "
+-                             "this kernel is not SELinux-enabled"));
++                             "it requires an SELinux-enabled kernel and root access"));
```

Comment:
Mega-nit: I don't know if this will ever actually bite anyone, but this looks like an unmodified error message and could be misleading (or misgoogling). Consider "this kernel is not SELinux-enabled, or you are using Termux and not running as root", or some similar appendage.